prac_05/word_occurences.py

```diff
@@ -8,3 +8,56 @@
 - Align the numbers so they are in one nice column (see prac for notes)
 """
 
+from operator import itemgetter
+
+
+def main():
+    # requests phrase from user
+    phrase = input("enter a phrase:")
+    new_dictionary = convert_to_dict(phrase)
+    longest_word = get_longest_word(new_dictionary)
+    sorting_list = initialize_sorting_list(new_dictionary)
```

Comment:
`new_dictionary` and `sorting_list` don't tell me what they store.

Reply:
i've renamed these both to be more clear. have committed and will push tonight. thanks :)